test/wallet-http-test.js

```diff
@@ -883,6 +883,18 @@ describe('Wallet HTTP', function() {
     assert.deepEqual(state, res);
   });
 
+  it('should get all wallet names', async () => {
+    const names = await wallet.client.get(`/wallet/${wallet.id}/name`);
+
+    assert.equal(names.length, 9);
+  });
+
+  it('should only get wallet-owned name resources', async () => {
+    const names = await wallet.client.get(`/wallet/${wallet.id}/name`, {own: true});
+
+    assert.equal(names.length, 3);
```

Comment:
This is ok but what would be really great is proof that these three names are the names this wallet won in auctions. One way to do this would be create an array `const ownedNames = []` up around line 50 with the other globals. Then in the `it` blocks where the wallet wins a name, `push` that name into the `ownedNames` array. Finally, down here in this `it` block, go through the array of `names` you get from the client and make sure the names you expect are in there.